pages/register.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { Formik, Field, FieldArray, Form, ErrorMessage } from 'formik';
+import superagent from 'superagent';
+
+import { businessDays } from '../utils/businessDays';
+import { businessHours } from '../utils/businessHours';
+import { instruments } from '../utils/instruments';
+
+export default function RegisterPage() {
+  async function handleSubmit(values) {
+    console.log(values);
+    // const response = await superagent.post(
+    //   'http://localhost:3000/api/register',
+    //   JSON.stringify(values)
+    // );
+    // console.log(response);
+  }
+  return (
+    <div>
+      <Formik onSubmit={handleSubmit}>
+        {({
```

Comment:
could we extract this outside of the return for cleaner code?

Reply:
Not really, Formik uses the render props pattern (either props.children or render={()=>{...}})

Source : [Formik documentation](https://jaredpalmer.com/formik/docs/api/formik)